src/components/HeaderView/HeaderView.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components";
+
+const HeadingWrapper = styled.header`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 2rem 0;
+  background-color: #04151f;
+`;
+
+const Heading = styled.h1`
```

Comment:
why is the style not in separate files?

Reply:
I used to put styles in this component file, but you're right

Reply:
But on the other hand, I think that placing styles in component's file is not considered a bad practice referring to the CSS-in-JS concept. Such solution is used in the styled-components documentation.

Reply:
IMO for tiny tiny projects it's fine, but then again tiny-tiny projects pay tiny-tiny wages, let's do it like grownup projects would.

Reply:
Let's agree on picking one, no reason to be opinionated just yet.